draft-wang-ppm-differential-privacy.md

```diff
@@ -1077,21 +1119,36 @@ privacy amplification by aggregation, assuming each Client has added the
 required amount of local DP noise, and there are at least minimum batch size
 number of Clients in the aggregation.
 
-In Aggregator DP settings, an Aggregator applies noise on the aggregation.
-Aggregator DP relies on the server being secure and trustworthy. Aggregators
-built using DAP protocol is ideal for this setting because DAP ensures no server
-can access any individual data, but only the aggregation.
+In Aggregator randomization settings, an Aggregator applies noise on the
+aggregation. This approach relies on the server being secure and trustworthy.
+Aggregators built using DAP protocol is ideal for this setting because DAP
+ensures no server can access any individual data, but only the aggregation.
 
 If there are no local DP added from client, noise added to the aggregation
 provides the privacy guarantee of the aggregation.
 
-One can use the Aggregator DP noise together with local DP noise to achieve
-privacy guarantee. If the DP guarantee is achieved with a minimum batch size
-number of Clients adding local DP noise, and minimum batch size is not reached
-when a data collection task expires, each Aggregator can add the remaining noise
-by generating the same local DP noise, on the missing number of Clients being
-the gap between actual number of Clients and minimum batch size.
+One can use the Aggregator randomization together with client randomization to
+achieve privacy guarantee. If the DP guarantee is achieved with a minimum batch
+size number of Clients adding local DP noise, and minimum batch size is not
+reached when a data collection task expires, each Aggregator can add the
+remaining noise by using the same Client randomizer, on the missing number of
+Clients being the gap between actual number of Clients and minimum batch size.
+
+## How to define neighboring batches {#neighboring-batch}
```

Comment:
Putting other definitions of neighboring batches into Appendix per Kunal's suggestion.